app/controllers/discussions_controller.rb

```diff
@@ -1,8 +1,9 @@
 class DiscussionsController < ApplicationController
 
-  before_action :set_event, only: [:index, :show, :new, :edit, :create, :update]
+  before_action :set_event, except: [:destroy]
```

Comment:
Okay, but its better to use `only` since it would prevent unintentional calling of method if you add any other action to controller